sherpa-onnx/csrc/offline-recognizer-whisper-impl.h

```diff
@@ -23,29 +23,32 @@
 
 namespace sherpa_onnx {
 
-static OfflineRecognitionResult Convert(const OfflineWhisperDecoderResult &src,
-                                        const SymbolTable &sym_table) {
-  OfflineRecognitionResult r;
-  r.tokens.reserve(src.tokens.size());
-
-  std::string text;
-  for (auto i : src.tokens) {
-    if (!sym_table.Contains(i)) {
-      continue;
+class OfflineRecognizerWhisperImpl : public OfflineRecognizerImpl {
+ private:
```

Comment:
@csukuangfj this is a private method, I will move it below after you take a look at the diffs.  I just converted the existing static method into this instance function.  The reason is that it needs access to ApplyInverseTextNormalization().